src/com/lftechnology/java/training/krishna/employeemanagement/controllers/LoginController.java

```diff
@@ -0,0 +1,476 @@
+
+		Scanner scanner = new Scanner(System.in);
+		authentiateUser(scanner);
+		scanner.close();
+
+	}
+
+	/**
+	 * This method is used to authenticate user
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void authentiateUser(Scanner scanner) {
+
+		showLoginTitle();
+		user = loginAction(scanner);
+		while (user == null) {
+			System.out.println("Invalid username and password. Please try again.\n");
+			loginAction(scanner);
+		}
+		showMainTitle(user.getUsername());
+		showMainMenu(scanner, user.getId());
+	}
+
+	/**
+	 * This method is used to display title of login
+	 *
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void showLoginTitle() {
+
+		System.out.println("\n"
+			+ "*******************************************************" + "\n"
+			+ "**                                                   **" + "\n"
+			+ "**              Login                                **" + "\n"
+			+ "**                                                   **" + "\n"
+			+ "*******************************************************" + "\n");
+	}
+
+	/**
+	 * This method is used to login using user information
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @return User
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static User loginAction(Scanner scanner) {
+
+		System.out.println("Please enter the username::");
+		String username = ValidationUtils.emptyValidation(scanner, "Username");
+		System.out.println("Please enter the password::");
+		String password = ValidationUtils.emptyValidation(scanner, "Password");
+		userServiceImpl = new UserServiceImpl();
+		user = userServiceImpl.login(username, password);
+		return user;
+	}
+
+	/**
+	 * This method is used to display username
+	 *
+	 * @param username
+	 *            {@link String}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void showMainTitle(String username) {
+
+		System.out.println("\n" +
+			"*******************************************************" + "\n" +
+			"**                                                   **" + "\n" +
+			"**              Welcome To " + username + "                   **" +
+			"\n" + "**                                                   **" +
+			"\n" + "*******************************************************" +
+			"\n");
+	}
+
+	/**
+	 * This method is used to display main menu
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @param id
+	 *            {@link int}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void showMainMenu(Scanner scanner, int id) {
+
+		employeeServiceImpl = new EmployeeServiceImpl();
+		Employee employee = employeeServiceImpl.findById(id);
+		if (employee.getRole().equalsIgnoreCase(Role.Admin.toString())) {
+			System.out.println(""
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)